docs/core_docs/docs/troubleshooting/errors/GRAPH_RECURSION_LIMIT.mdx

```diff
@@ -0,0 +1,14 @@
+# GRAPH_RECURSION_LIMIT
+
+Your LangGraph app reached the maximum number of steps before hitting a stop condition.
+
+This is usually a sign of an infinite loop, but complex graphs may hit this naturally.
```

Comment:
should we have an example repro of how to trigger this error for mental pattern matching?